src/workerd/api/global-scope.h

```diff
@@ -364,12 +364,8 @@ class ServiceWorkerGlobalScope: public WorkerGlobalScope {
     return jsg::alloc<Performance>();
   }
 
-  jsg::Unimplemented getOrigin() { return {}; }
-  // TODO(conform): A browser-side service worker returns the origin for the URL on which it was
-  //   installed, e.g. https://www.example.com for a service worker downloaded from
-  //   https://www.example.com/sw.js. This seems like something we could provide to scripts during
-  //   a FetchEvent callback (in between .request() and FetchEvent.respondWith()), and otherwise
-  //   throw.
+  kj::StringPtr getOrigin() { return "null"; }
+  // The origin is unknown, return "null" as advocated in https://developer.mozilla.org/en-US/docs/Web/API/origin.
```

Comment:
Likely worth referencing the spec here directly: https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-opaque

Reply:
Thanks, done in https://github.com/cloudflare/workerd/pull/622/commits/b12b7dab9e4a9869b4a88d7fad5fd6001e19104c